src/redux/books/books.js

```diff
@@ -1,25 +0,0 @@
-// eslint-disable-next-line
-import { v4 as uuidv4 } from 'uuid';
-
-const booksArray = [
```

Comment:
[OPTIONAL] In categories I'm having an issue with the check status button. Because it's not showing me the status that the user wants to see. Kindly make it functional as well. Thank you!